contracts/axelar-gas-service/tests/test.rs

```diff
@@ -369,7 +358,9 @@ fn refund() {
 
     let message_id = message_id(&env);
 
-    client.refund(&message_id, &receiver, &token);
+    client
+        .mock_all_auths()
```

Comment:
this is checking for success, so auth should be asserted as well. Check if this applies in other tests too